test/clj/rems/application/test_search.clj

```diff
@@ -1,30 +1,104 @@
 (ns ^:integration rems.application.test-search
   (:require [clojure.test :refer :all]
             [rems.application.search :as search]
+            [rems.db.applications :as applications]
             [rems.db.test-data :as test-data]
-            [rems.db.testing :refer [test-db-fixture rollback-db-fixture]]))
+            [rems.db.testing :refer [test-db-fixture search-index-fixture]]))
 
-(use-fixtures :once test-db-fixture)
-(use-fixtures :each rollback-db-fixture)
+(use-fixtures :once (join-fixtures [test-db-fixture
+                                    search-index-fixture]))
```

Comment:
rollback not needed?

Reply:
The rollback doesn't affect the search index, so it's better to not pretend that there is any rollback. It would only create weird error conditions if a second deftest were added.